Management/src/ActuatorWeb/appsettings.Development.json

```
@@ -0,0 +1,41 @@
+{
+  // Steeltoe: Add schema to get auto-completion.
+  "$schema": "https://steeltoe.io/schema/latest/schema.json",
+  "DetailedErrors": true,
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  // Steeltoe: Configure exposure of all actuators.
+  "Management": {
+    "Endpoints": {
+      "Actuator": {
+        "Exposure": {
+          "Include": [
+            "*"
+          ]
+        }
+      }
+    }
+  },
+  // Steeltoe: Configure client for Spring Boot Admin.
+  "Spring": {
+    "Boot": {
+      "Admin": {
+        "Client": {
+          // For podman compatibility, change host.docker.internal to host.containers.internal
+          "BasePath": "http://host.docker.internal:8090",
+          "Metadata": {
+            "user.name": "actuatorUser",
+            "user.password": "actuatorPassword"
+          },
+          "Url": "http://localhost:9099"
+        }
+      }
+    }
+  },
+  // Steeltoe: register this application with a Spring Boot Admin server.
```

```suggestion
  // Steeltoe: Register this application with a Spring Boot Admin server.
```
sync up with the other project